py/make_day_requests.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+trap "kill 0" EXIT
+which parallel &> /dev/null
+if [ $? != 0 ]; then
+        echo "parallel is required please install it"
+        echo "sudo apt-get install parallel"
+        exit 1
+fi
+set -e
+
+function usage {
+  echo -e "Usage:\n-f day file to process\n" 1>&2
+  echo "Example: AWS_DEFAULT_PROFILE=opentraffic $0 -f 2017_01_02.gz -b localhost:9092 -t raw" 1>&2
```

Comment:
Need to update Usage to reflect actual args (e.g., AWS_DEFAULT_PROFILE not needed)